package/Runtime/Scripts/Internal/NimbusAdUnit.cs

```diff
@@ -130,11 +130,12 @@ public sealed class NimbusAdUnit {
 				case AdEventTypes.COMPLETED:
 					_adEvents.EmitOnOnVideoAdCompleted(this);
 					break;
+				case AdEventTypes.SKIPPED:
+					if (AdType == AdUnityType.Rewarded) {
+					    _adEvents.EmitOnOnVideoAdSkipped(this);
+					}
+					break;
```

Comment:
See the above, this needs a better pathway. This isn't an event that will be exposed to any public API. This is a hack to determine if the Ad was skipped. This does not need event propagation. Likely a boolean flag on the ad object itself is all is that needed to be set.